iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractNodeBreakerTest.java

```diff
@@ -367,7 +367,7 @@ public void errorDisconnectOnRemoved() {
         l1.getTerminal().disconnect(SwitchPredicates.IS_CLOSED_BREAKER);
 
         // remove load
-        l1.remove();
+        l1.remove();    // <---- TODO HERE
```

Comment:
This test passes on our side thanks to https://github.com/powsybl/powsybl-network-store/blob/a3bffa4b1e9fdf15c5dc379080be3dc9a5fd6821/network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/TerminalImpl.java#L393

Reply:
But do we allow l1.getTerminal() on a removed line